src/transformers/models/speecht5/configuration_speecht5.py

```diff
@@ -168,8 +168,6 @@ class SpeechT5Config(PretrainedConfig):
             The maximum sequence length of text features that this model might ever be used with.
         encoder_max_relative_position (`int`, *optional*, defaults to 160):
             Maximum distance for relative position embedding in the encoder.
-        decoder_max_relative_position (`int`, *optional*, defaults to 160):
-            Maximum distance for relative position embedding in the dencoder.
```

Comment:
@hollance Could you double check this unused attribute? Does it make sense to remove it?

Reply:
I think this may be used by the ForPreTraining version of this model but we don't currently support this, so yes it can be removed.